Zappy_IA_Report/DTC/Video/Tutorial1.py

```
@@ -0,0 +1,19 @@
+import pandas as pd
+from skelarn.model_selection import train_test_split
```

Pylint (3.3.7)

[error] Unable to import 'skelarn.model_selection'

(E0401)